<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xperia Kernel Tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/res/main.ico</normaloff>:/icons/res/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="bootImage">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/icons/res/plus.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="statusLog">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Boot_Image"/>
   <addaction name="actionSave_Boot_Image"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Boot_Image"/>
    <addaction name="actionSave_Boot_Image"/>
    <addaction name="separator"/>
    <addaction name="actionSplit_BootImage"/>
    <addaction name="actionMake_Boot_Image"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionExtract_Components"/>
    <addaction name="actionAlways_Confirm_Device_Resolution"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionSplit_BootImage">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/res/bomb.png</normaloff>:/icons/res/bomb.png</iconset>
   </property>
   <property name="text">
    <string>Split BootImage</string>
   </property>
  </action>
  <action name="actionMake_Boot_Image">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/res/fw.png</normaloff>:/icons/res/fw.png</iconset>
   </property>
   <property name="text">
    <string>Make Boot Image</string>
   </property>
  </action>
  <action name="actionOpen_Boot_Image">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/res/plus.ico</normaloff>:/icons/res/plus.ico</iconset>
   </property>
   <property name="text">
    <string>Open Boot Image</string>
   </property>
  </action>
  <action name="actionExtract_Components">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/res/conf.ico</normaloff>:/icons/res/conf.ico</iconset>
   </property>
   <property name="text">
    <string>Extract Components</string>
   </property>
  </action>
  <action name="actionSave_Boot_Image">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/res/reactor.png</normaloff>:/icons/res/reactor.png</iconset>
   </property>
   <property name="text">
    <string>Save Boot Image</string>
   </property>
  </action>
  <action name="actionAlways_Confirm_Device_Resolution">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always Confirm Device Resolution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
